[2022-08-06 19:38:37,048] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-11T00:00:00+00:00 [queued]>
[2022-08-06 19:38:37,067] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-11T00:00:00+00:00 [queued]>
[2022-08-06 19:38:37,067] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:37,067] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:38:37,067] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:37,094] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-04-11 00:00:00+00:00
[2022-08-06 19:38:37,102] {standard_task_runner.py:52} INFO - Started process 2513 to run task
[2022-08-06 19:38:37,105] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-04-11T00:00:00+00:00', '--job-id', '771', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp2jq5aj6x', '--error-file', '/tmp/tmp1zyem_6t']
[2022-08-06 19:38:37,105] {standard_task_runner.py:80} INFO - Job 771: Subtask second_task
[2022-08-06 19:38:37,180] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-11T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:38:37,289] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-11T00:00:00+00:00
[2022-08-06 19:38:37,290] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:38:37,291] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:38:37,309] {subprocess.py:85} INFO - Output:
[2022-08-06 19:38:37,311] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:38:37,311] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:38:37,346] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220411T000000, start_date=20220806T193837, end_date=20220806T193837
[2022-08-06 19:38:37,396] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:38:37,432] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:52:28,549] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-11T00:00:00+00:00 [queued]>
[2022-08-06 21:52:28,564] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-11T00:00:00+00:00 [queued]>
[2022-08-06 21:52:28,564] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:28,564] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:52:28,564] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:28,584] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-04-11 00:00:00+00:00
[2022-08-06 21:52:28,592] {standard_task_runner.py:52} INFO - Started process 2706 to run task
[2022-08-06 21:52:28,594] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-04-11T00:00:00+00:00', '--job-id', '772', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpluq7c4iu', '--error-file', '/tmp/tmp80enb66g']
[2022-08-06 21:52:28,595] {standard_task_runner.py:80} INFO - Job 772: Subtask second_task
[2022-08-06 21:52:28,672] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-11T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:52:28,772] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-11T00:00:00+00:00
[2022-08-06 21:52:28,774] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:52:28,774] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:52:28,791] {subprocess.py:85} INFO - Output:
[2022-08-06 21:52:28,793] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:52:28,793] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:52:28,828] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220411T000000, start_date=20220806T215228, end_date=20220806T215228
[2022-08-06 21:52:28,886] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:52:28,920] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:57:56,148] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-11T00:00:00+00:00 [queued]>
[2022-08-06 21:57:56,162] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-11T00:00:00+00:00 [queued]>
[2022-08-06 21:57:56,163] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:57:56,163] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:57:56,163] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:57:56,184] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-04-11 00:00:00+00:00
[2022-08-06 21:57:56,192] {standard_task_runner.py:52} INFO - Started process 2513 to run task
[2022-08-06 21:57:56,194] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-04-11T00:00:00+00:00', '--job-id', '774', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpos80x2la', '--error-file', '/tmp/tmpu6rneaa0']
[2022-08-06 21:57:56,194] {standard_task_runner.py:80} INFO - Job 774: Subtask second_task
[2022-08-06 21:57:56,288] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-11T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:57:56,421] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-11T00:00:00+00:00
[2022-08-06 21:57:56,422] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:57:56,423] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:57:56,442] {subprocess.py:85} INFO - Output:
[2022-08-06 21:57:56,446] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:57:56,447] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:57:56,499] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220411T000000, start_date=20220806T215756, end_date=20220806T215756
[2022-08-06 21:57:56,528] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:57:56,578] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:12:56,440] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-11T00:00:00+00:00 [queued]>
[2022-08-06 22:12:56,462] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-11T00:00:00+00:00 [queued]>
[2022-08-06 22:12:56,463] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:12:56,463] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:12:56,463] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:12:56,483] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-04-11 00:00:00+00:00
[2022-08-06 22:12:56,490] {standard_task_runner.py:52} INFO - Started process 2511 to run task
[2022-08-06 22:12:56,493] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-04-11T00:00:00+00:00', '--job-id', '773', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp6d4q157v', '--error-file', '/tmp/tmp33f2s718']
[2022-08-06 22:12:56,493] {standard_task_runner.py:80} INFO - Job 773: Subtask second_task
[2022-08-06 22:12:56,591] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-11T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:12:56,700] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-11T00:00:00+00:00
[2022-08-06 22:12:56,703] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:12:56,704] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:12:56,727] {subprocess.py:85} INFO - Output:
[2022-08-06 22:12:56,727] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:12:56,727] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:12:56,801] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220411T000000, start_date=20220806T221256, end_date=20220806T221256
[2022-08-06 22:12:56,824] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:12:56,864] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:24:14,064] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-11T00:00:00+00:00 [queued]>
[2022-08-06 22:24:14,077] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-11T00:00:00+00:00 [queued]>
[2022-08-06 22:24:14,077] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:24:14,077] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:24:14,077] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:24:14,090] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-04-11 00:00:00+00:00
[2022-08-06 22:24:14,096] {standard_task_runner.py:52} INFO - Started process 2553 to run task
[2022-08-06 22:24:14,098] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-04-11T00:00:00+00:00', '--job-id', '773', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp_fdts5gw', '--error-file', '/tmp/tmpd61242v_']
[2022-08-06 22:24:14,099] {standard_task_runner.py:80} INFO - Job 773: Subtask second_task
[2022-08-06 22:24:14,153] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-11T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:24:14,234] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-11T00:00:00+00:00
[2022-08-06 22:24:14,235] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:24:14,235] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:24:14,246] {subprocess.py:85} INFO - Output:
[2022-08-06 22:24:14,247] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:24:14,247] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:24:14,270] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220411T000000, start_date=20220806T222414, end_date=20220806T222414
[2022-08-06 22:24:14,310] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:24:14,337] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
